src/test/resources/results/scriptCommand/testScriptCommandForSchemaEvolution_Alter_Table_Modify_column/Makefile-configDiff

```diff
@@ -0,0 +1,6 @@
+
```

Comment:
https://github.com/phdata/streamliner/pull/43/commits/cd9151e8a2be5766c44b1165c16d04a807e7b440#diff-42a08f7e94f27f53e3401539ad8346c614c99d134172c127d8bf8f84a48d2e54
This file is empty because this is case of modify column where we are throwing exception with number of errors to investigate